Documentation/api.md

```diff
@@ -177,7 +177,7 @@ BasicAuth allow an endpoint to authenticate over basic authentication More info:
 
 ## Endpoint
 
-Endpoint defines a scrapeable endpoint serving Prometheus metrics.
+Endpoint defines a scrapable endpoint serving Prometheus metrics.
```

Comment:
This is actually generated docs, hence the failed tests. The typo lives here https://github.com/coreos/prometheus-operator/blob/b0c8b4da94276df08d3fe1137d05f3e8b7bdf61d/example/prometheus-operator-crd/servicemonitor.crd.yaml 

After its fixed in the CRD https://github.com/coreos/prometheus-operator#contributing should help with generating these docs files. :) 

Same goes for the other typo below.

Reply:
Nice, CI is actually showing this nicely - However, I think I would suggest failing further CI job once a job fails. What do you think?

Reply:
(not in this PR)

Reply:
To fail all of them? hmm not sure I agree, as I like the idea of each being independent, at least this one. I agree on vendor, or go compiler errors, but for docs I like it being isolated.

Reply:
So actually https://github.com/coreos/prometheus-operator/pull/2764#discussion_r325731231 is not true ;p The source of true was in go file. I update docs to make sure it's clear.

Reply:
> To fail all of them? hmm not sure I agree, as I like the idea of each being independent, at least this one. I agree on vendor, or go compiler errors, but for docs I like it being isolated.

Problems:
* Waste of Travis worker - we have limited one, so probably we delay other builds
* Build is still pending as other jobs are being done, however we want to know as quick as possible that we need to fix the PR ... so why waiting with red failed mark in Github until all jobs are done if the first fails?

Anyway, small nit (:

Reply:
yes sorry too many things at once :) the go is the correct place yes!

> Waste of Travis worker - we have limited number of those (if not one) for OSS, so probably we delay other builds

yes I agree, I guess it depends how you look at it. I am fine with changing it to that, but we haven't had huge delays so far with builds so don't see a need.

Reply:
Unless we have some measurable impact I'm not sure how to even evaluate if cancelling other jobs would even have effect. It doesn't seem like travis supports something like this and we rarely have multiple jobs running at once so I don't feel this is our highest priority.

Reply:
Sure this was a side note (: